ompi/include/mpi.h.in

```diff
@@ -521,13 +521,6 @@ enum {
     MPI_WIN_DISP_UNIT,
     MPI_WIN_CREATE_FLAVOR,
     MPI_WIN_MODEL,
-
```

Comment:
while probably not practically important, it is worth noting that this is technically an API break.  Not that anyone could ever do anything with the API components being removed.

Reply:
Fair point.

Should I put a "Target 5.0" label on here, and perhaps make a new label for "ABI break"?